tasks/cleanup-workspace/cleanup-workspace.yaml

```diff
@@ -15,13 +15,16 @@ spec:
     - name: subdirectory
       type: string
       description: The directory to remove within the workspace
+    - name: delay
+      type: string
+      default: 60
+      description: time in seconds to delay the cleanup action
```

Comment:
Nit: The string should start with a capital: `Time in ...`

Reply:
Fixed